### ISSUE TITLE
Finnish menu translation typos and extraneous French block?

### ISSUE THREAD

**Issue by oljemark** [vim/vim#16285]
**Is your feature request about something that is currently impossible or hard to do? Please describe the problem.**
I found two typos in https://github.com/vim/vim/blob/master/runtime/lang/menu_fi_fi.latin1.vim#L91 . I also checked the Accel keys given on line 42: https://github.com/vim/vim/blob/master/runtime/lang/menu_fi_fi.latin1.vim#L42 : looking at the following lines and ignoring a submenu, I only get as far as KPTLOIEJSAHRUKY out of a given key collection KPTLOIEJSAHRUYKVNF - have some menu items been deleted? Looking at the git blame here I don't see anything after history starts at commit 8c8de839325eda0bed68917d18179d2003b344d1 in version 7.2a. 
Likewise, the two French blocks between lines L220-L318 (https://github.com/vim/vim/blob/master/runtime/lang/menu_fi_fi.latin1.vim#L220) & L459-L464 haven't been changed since then - why are they in the Finnish translation?

**Comment by flammie** [comment]
Typo fixes are definitely in order, cannot remember touching that file even, I wonder if there's been some semi-automatic updating gone wrong back in the day, fi and fr are quite close as language codes. Perhaps there was a translator mailing list back in the day we used to update translations that has relevant archive?

**Comment by oljemark** [comment]
Hi, @flammie . Looking at the vim/vim-history repository it seems the finnish menu translation was introduced after version 6.4 but before 7.2.

**Comment by oljemark** [comment]
FYI @flammie . 

I looked for some such relevant mailing list archive, but did not find it:
Since I saw you had originally (?) added the menu translation 4.9.2007, I looked at the Google group vim_dev /Gmane vim.devel groups between 2006-2008, but it seems the translations were not tracked in the same CVS repository as the code, see Bram's 2nd answer here (https://groups.google.com/g/vim_dev/search?q=vim7%2Fsrc%2Fpo%2Fsv.po). The only full menu translation I found posted in that group was an update for 7.0 of menu_br_pt.vim, posted 19.9.2006. 

The full block of French comes from here: https://github.com/vim/vim/blame/master/runtime/lang/menu_fr_fr.latin1.vim#L374 , dated 2004, before your translation - maybe you started working on yours by modifying the french menu translation?
